Gemfile

```diff
@@ -70,3 +70,6 @@ group :test do
 end
 
 gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
+
+# Rack middleware for blocking & throttling abusive requests
+gem 'rack-attack'
```

Comment:
this should be under the `# WebServer` section of the gemfile

Reply:
Done 🙌